packages/story-editor/src/components/panels/design/textStyle/panelHeader.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@web-stories-wp/i18n';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  Icons,
+  themeHelpers,
+} from '@web-stories-wp/design-system';
+import { useContext } from '@web-stories-wp/react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { trackEvent } from '@web-stories-wp/tracking';
+import styled from 'styled-components';
+import useApplyTextAutoStyle from '../../../../utils/useApplyTextAutoStyle';
+import Tooltip from '../../../tooltip';
+import PanelTitle from '../../panel/shared/title';
+import panelContext from '../../panel/context';
+
+const AutoStyleButton = styled(Button).attrs({
+  variant: BUTTON_VARIANTS.SQUARE,
+  type: BUTTON_TYPES.TERTIARY,
+  size: BUTTON_SIZES.SMALL,
+})`
+  margin-right: 4px;
+  ${({ theme }) =>
+    themeHelpers.focusableOutlineCSS(theme.colors.border.focus, '#1d1f20')};
+`;
+
+function PanelHeader({ selectedElements, pushUpdate }) {
+  const {
+    state: { isCollapsed },
+  } = useContext(panelContext);
+  const applyTextAutoStyle = useApplyTextAutoStyle(
+    selectedElements[0],
+    (textProps) => {
+      pushUpdate(textProps, true);
+      trackEvent('auto_style_text');
+    }
+  );
+
+  const getActions = () => {
+    if (isCollapsed) {
+      return null;
+    }
+    return (
+      <Tooltip title={__('Adaptive text colors', 'web-stories')}>
+        <AutoStyleButton
+          aria-label={__('Adaptive text colors', 'web-stories')}
+          onClick={applyTextAutoStyle}
+          disabled={selectedElements?.length > 1}
+        >
+          <Icons.ColorBucket />
+        </AutoStyleButton>
```

Comment:
This icon is slightly off the line here - should have less right-margin so it's aligned with the elements inside the panels:

![Screen Shot 2021-11-01 at 16 29 00](https://user-images.githubusercontent.com/637548/139741435-3f5526d5-2bb7-40cc-a1c1-d31ec14b4b6f.png)